### PR TITLE
Add /sys/kernel/debug VolumeMount to helm chart, when eBPF is in use

### PR DESCRIPTION
**Pull request by jcpittman144**
## What this PR does / why we need it:
Agent eBPF code requires access to the /sys/kernel/debug filesystem.
For certain kernel versions, this filesystem is not accessible by default
from within docker containers, so add a VolumeMount in the helm chart.

## Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x ] PR only contains changes for one chart

### CODE REVIEW

File: charts/sysdig/templates/daemonset.yaml [hunk 1]
```diff
@@ -229,6 +233,9 @@ spec:
         {{- if (and .Values.ebpf.enabled .Values.slim.enabled) }}
         - name: bpf-probes
           emptyDir: {}
+	- name: sys-tracing
+          hostPath:
+	    path: /sys/kernel/debug
```
Comment (airadier) [inline charts/sysdig/templates/daemonset.yaml:238]: It looks there is some weird indentation here? Maybe tabs vs spaces?

**Review by airadier — CHANGES_REQUESTED** [review]

**Comment by airadier** [comment]
Thanks for the PR! The linter is failing because chart version was not updated un chart.yaml. Could you please update that and rhe CHANGELOG.md file? I noticed some identation issues that might need fixing too.

File: charts/sysdig/templates/daemonset.yaml (hunk 1 above)
Comment (jcpittman144) [reply charts/sysdig/templates/daemonset.yaml:238]: Thanks for the review.  I will fix the indentation, update the version and CHANGELOG, and resubmit the PR.
Thanks!

**Comment by jcpittman144** [comment]
Closing this PR, rebasing against an updated version of the repo.

**Pull request closed without merging**